components/engine_test/src/lib.rs

```diff
@@ -244,9 +244,19 @@ pub mod kv {
 
     // Extract tablet id and tablet suffix from the path.
     fn get_id_and_suffix_from_path(path: &Path) -> (u64, u64) {
+        let split_path = path
+            .as_os_str()
+            .to_str()
+            .map_or(false, |s| s.contains("split"));
```

Comment:
> contains
Should it be strictly prefix? contains may match the string token in the middle of the path.